locales/en.json

```diff
@@ -152,7 +152,9 @@
       "loading": "Loading {resource}",
       "selected": "{selectedItemsCount} selected",
       "allItemsSelected": "All {itemsLength}+ {resourceNamePlural} in your store are selected.",
+      "allFilteredItemsSelected": "All {itemsLength}+ {resourceNamePlural} in this filter are selected.",
```

Comment:
:bike: 🏠 `...on this filter in your store are selected` ?

If only to make sure merchant's are aware it's all customers in their store

Reply:
i think what we have is fine personally, but i don't have a super strong opinion. this label only comes up when somebody clicks directly the button that says `Select all XY+ resources in this filter`, so i feel that they know what they've selected at this point 😁

Reply:
I would keep what you have which matches the custom fork `ResourceListWithHeader` used in Products.
`All {itemsLength}+ {resourceNamePlural} in this filter are selected.`